service/src/test/java/uk/gov/hmcts/reform/fpl/service/CafcassEmailContentProviderTest.java

```diff
@@ -88,4 +105,60 @@ void shouldReturnSuccessfullyWithEmptyCaseDetails() throws IOException {
         assertThat(cafcassEmailContentProvider.buildCafcassSubmissionNotification(emptyCaseDetails(),
             LOCAL_AUTHORITY_CODE)).isEqualTo(expectedMap);
     }
+
+    @Test
+    void shouldReturnExpectedMapWithValidSDODetails() throws IOException {
```

Comment:
there's a lot of setup for this test. Could you extract anything to separate private methods potentially?